src/impl/generated_specializations_cpp/abs/KokkosBlas1_abs_eti_spec_inst_Kokkos_complex_double__LayoutLeft_Serial_HostSpace.cpp

```diff
@@ -1,58 +0,0 @@
-/*
```

Comment:
Bunch of these deletions. I am not going to comment on each one of them, but it would be good to know why this is showing as deleted.